apps/frontend/app/(website)/studio/features/FirstLoginExperience/config.tsx

```diff
@@ -4,22 +4,22 @@ const steps = [
   {
     element: "codemod",
     description:
-      "<p>This section is dedicated to codemods.</p><p>Codemods are tools used to automate code modifications and refactorings. They allow developers to make large-scale changes across a codebase quickly and consistently.</p><p> Here, you can see an example codemod script that transforms code from a 'Before' state to an 'After' state using jscodeshift.</p><p> You can modify the codemod's code here.</p>",
+      "<p>This section is dedicated to codemods.</p><p>Codemods are tools used to automate code modifications and refactorings. They allow developers to make large-scale changes across a codebase quickly and consistently.</p><p>The codemod editor allows you to see and modify the codemod you are building.</p>",
   },
   {
     element: "before-and-after-panels",
     description:
-      "<p>These panels display the code before and after applying a codemod. The 'Before' panel shows the original code, while the 'After' panel displays the expected output after transformation.</p><p> This visual comparison helps users understand the changes made by the codemod, ensuring the modifications meet the desired outcome.</p>",
+      "<p>The before/after panels allow you to specify how the code should look like before and after transformation.</p><p>To get started, insert your current code into the 'Before' panel, and how you want your code to look like after running the codemod into the 'After' panel.</p>",
   },
   {
     element: "assistant",
     description:
-      "<p>The assistant section is available for logged users only and provides support and guidance throughout the codemod creation and execution process.</p> <p>It leverages AI to help users generate codemod scripts, troubleshoot issues, and refine their code transformations. This assistant is an invaluable resource for both novice and experienced developers looking to optimize their coding workflow. In the 'Debug' tab the codemods errors are listed and output of the console (<code>console.log</code> in codemod editor can be used)</p>",
+      "<p>The AI assistant provides support and guidance throughout the codemod creation and execution process by generating codemod scripts, troubleshooting issues, and incrementally refining code transformations.</p><p>The 'Debug' tab lists the codemod workflow and potential errors.</p>",
```

Comment:
too many not so much helpful words... instead:

"Assistant section includes AI, and for advanced users: AST and debugger. 
AI requires login. Start by clicking the "Generate codemod with AI" button."